features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/signicat/metadata.json

```
@@ -0,0 +1,272 @@
+                        "ariaLabel": "Client ID", 
+                        "type": "text", 
+                        "name": "clientId", 
+                        "label": "Client ID",
+                        "placeholder": "Enter Client ID.",
+                        "required": true, 
+                        "autoComplete": true,
+                        "data-testid": "signicat-idp-create-wizard-content-idp-client-id",
+                        "maxLength": "100",
+                        "minLength": "3",
+                        "width": "13"
+                    },
+                    {
+                        "index": 2,
+                        "ariaLabel": "Client Secret",
+                        "className": "addon-field-wrapper",
+                        "type": "password", 
+                        "name": "clientSecret", 
+                        "label": "Client secret",
+                        "placeholder": "Enter Client secret.",
+                        "required": true, 
+                        "autoComplete": true,
+                        "data-testid": "signicat-idp-create-wizard-content-idp-client-secret",
+                        "maxLength": "100",
+                        "minLength": "3",
+                        "width": "13"
+                    }
+                ]
+                
+            },
+            "wizardHelp": {
+                "message": {
+                    "copyInputFields": [
+                        {
+                            "description": "Use the following URL as the <strong>Authorized Redirect URI</strong>.",
+                            "value": "redirect_uri"
+                        }
+                    ],
+                    "header": "Prerequisites",
+                    "paragraphs": [
+                        "Before you begin, create an <strong>OIDC credential</strong> on the <a href=\"https://dashboard.signicat.com\" target=\"_blank\" rel=\"noopener noreferrer\">Signicat Dashboard</a>, and obtain a <strong>Client ID & secret</strong>."                    ],
+                    "link": {
+                        "text": "See Signicat's guide on configuring OIDC Apps.",
+                        "url": "https://developer.signicat.com/docs/eid-hub/"
+                    }
+                },
+                "fields": [
+                    {
+                        "fieldName": "Name",
+                        "hint": "Provide a unique name for the connection."
+                    },
+                    {
+                        "fieldName": "Client ID",
+                        "hint": "Provide the <Code class=\"inline-code\">Client ID</Code> you received from Signicat when you registered the OIDC app."
+                    },
+                    {
+                        "fieldName": "Client secret",
+                        "hint": "Provide the <Code class=\"inline-code\">Client secret</Code> you received from Signicat when you registered the OAuth app."
```

```suggestion
                        "hint": "Provide the <Code class=\"inline-code\">Client secret</Code> you received from Signicat when you registered the OIDC app."
```
I've noticed that all other places mention this as `OIDC app`.

---

fixed